docs/LoongArch-ELF-ABI-EN.adoc

```diff
@@ -628,3 +628,197 @@ with check 32-bit unsigned overflow
 |Linux, Glibc
 |/lib32/ld-linux-loongarch-ilp32s.so.1
 |===
+
+== Procedure Calling Convention
+
+The basic principle of the LoongArch procedure calling convention is to pass arguments in registers as much as possible (i.e. floating-point arguments are passed in floating-point registers and non floating-point arguments are passed in general-purpose registers, as much as possible); arguments are passed on the stack only when no appropriate register is available.
+
+The argument registers are:
+
+. Eight floating-point registers `fa0-fa7`, and `fa0-fa1` are also used to return values. They are all used to pass floating-point arguments.
+
+. Eight general-purpose registers `a0-a7`, with `a0-a1` reused to return values.
+
+Generally, the GARs are used to pass fixed-point arguments, and floating-point arguments when no FAR is available.
+Bit fields are stored in little endian.
+In addition, subroutines should ensure that the values of general-purpose registers `s0-s9` and floating-point registers `fs0-fs7` are preserved across procedure calls.
+
+==== Note
+
+In this document, *GRLEN* is the bit width of general-purpose register, *FRLEN* is the bit width of float-point register and *WOA* is the bit width of the argument.
+The argument registers are collectively denoted as *AR* (*GAR* or *FAR*), where the general-purpose argument register is denoted as *GAR* and the floating-point argument register is denoted as *FAR*.
```

Comment:
Does using *AR* abbreviation justified? It is used in just one place and does not save too much typing, but clearly hide meaning, as all abbreviations. If you look for example at RISC-V document https://riscv.org/wp-content/uploads/2015/01/riscv-calling.pdf they are not abbreviate "integrer registers" and "floating-point registers" . Same for other ABI documents 
- https://github.com/hjl-tools/x86-psABI/wiki/x86-64-psABI-1.0.pdf
- https://github.com/ARM-software/abi-aa/
and others. 
So other question, does *GAR* and *FAR* abbreviation is justified?